internal/compiler/compiler_test.go

```diff
@@ -102,6 +102,15 @@ func TestCompilerSmoke(t *testing.T) {
 	err = compiler.BuildAll(ctx)
 	c.Assert(err, qt.IsNil)
 
+	// Verify created files/folders are as expected
+	// Look for existence of /2021-06-01~experimental
+	_, err = os.Stat(outputPath + "/2021-06-01~experimental")
+	c.Assert(err, qt.IsNil)
+
+	// Look for absence of /2021-06-01
+	_, err = os.Stat(outputPath + "/2021-06-01")
+	c.Assert(err, qt.Not(qt.IsNil))
```

Comment:
A little more precise:

```suggestion
	c.Assert(os.IsNotExist(err), qt.IsTrue)
```